public/locales/el/translation.json

```
@@ -31,6 +31,19 @@
       "mismatch": "Οι κωδικοί δεν είναι ίδιοι!"
     }
   },
+  "profile": {
+    "add-bio": "Γράψε κάτι για εσένα!",
+    "attended": "Δράσεις που έχετε συμμετάσχει",
```

We're using singular in the other phrases, why plural here?
```suggestion
    "attended": "Δράσεις που έχεις συμμετάσχει",
```